src/guidelines/a11y.md

```diff
@@ -0,0 +1 @@
+TBD
```

Comment:
@bwalker8038 I've got some stuff for this I can add but definitely would like you to piggyback with your expertise sir!